dcpy/test/extract/test_utils.py

```diff
@@ -0,0 +1,18 @@
+from dcpy.connectors.edm import recipes
+from dcpy.extract import utils
+
+
+class TestSocrata:
+    ## https://data.cityofnewyork.us/Social-Services/Outcome-of-Preventive-Cases-Closed-By-Borough-And-/q663-gvx6/about_data
+    ## this dataset has "no plan for updates"
+    ## it may eventually be deleted but for now is useful as a psuedo-constant
+    test_set = recipes.ExtractConfig.Source.Socrata(
+        type="socrata",
+        org=recipes.ExtractConfig.Source.Socrata.Org.nyc,
+        uid="q663-gvx6",
```

Comment:
Not sure what dataset this is but assuming it's not likely to change over time (uid and version)

Reply:
It is not - per the comment a few lines up, there are "no plan for updates" for this dataset. Have tweaked this slightly in the latest commit (and moved this test to `socrata` connector)